student-library/src/main/java/tds/student/sql/data/OpportunityItem.java

```diff
@@ -43,12 +37,13 @@
   private boolean _isVisible;
   private boolean _isPrintable;
   private ReturnStatus _returnStatus = null;
-  
+
   @JsonProperty ("BankKey")
   public long getBankKey () {
     return _bankKey;
   }
 
+  @JsonProperty("BankKey")
```

Comment:
You're correct.  I don't need these.  Closing the PR.  I fought these objects quite a bit and this is left over from that fight.